tests/test_align.py

```diff
@@ -0,0 +1,250 @@
+import pytest
```

Comment:
Please allow me to wait to comment on your testing code until the next iteration of feedback. I think there are a few changes to the function definitions, so I'll provide detailed feedback on the testing once those are solidified.

Reply:
Sounds good! Thank you again for the feedback.